script/lib/filesystem.py

```diff
@@ -51,15 +51,17 @@ def download_and_extract(destination, url):
     with contextlib.closing(urllib2.urlopen(url)) as u:
       total = int(u.headers['content-length'])
       done = 0
+      last_length = 0
       while True:
         chunk = u.read(1024)
         done += len(chunk)
         if not len(chunk):
           break
-        sys.stderr.write('                                   \r')
-        percent = str(round(float(done) / float(total), 4) * 100) + '%'
+        percent = '{0:.2f}%%'.format(round(float(done) / float(total), 4) * 100)
```

Comment:
@MarshallOfSound I just realized there is an extra `%` here, which ends up producing output like this:
```
-> 0.10%% (1MB/1729MB)
```
(From https://circleci.com/gh/electron/libchromiumcontent/8785?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)